Frends.Excel.Parse/Frends.Excel.Parse/Definitions/Result.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System.ComponentModel;
+using System.Data;
+
+namespace Frends.Excel.Parse.Definitions;
+
+public class Result
+{
+    /// <summary>
+    /// False if conversion fails.
+    /// </summary>
+    [DefaultValue("false")]
+    public bool Success { get; set; }
+
+    /// <summary>
+    /// Exception message.
+    /// </summary>
+    [DefaultValue("")]
+    public string ErrorMessage { get; private set; }
+
+    /// <summary>
+    /// Parsed Excel data set.
+    /// </summary>
+    /// <returns>String</returns>
+    public DataSet DataSet { get; set; }
```

Comment:
Same here?

Reply:
Setting this to internal to allow for altering from within the assebly.